docs/install/reference.md

```diff
@@ -22,15 +22,21 @@ Prints out the version of the binary and exits.
 
 ### Description
 
-Runs the Argo CD Image Updater, possibly in an endless loop.
+Runs the Argo CD Image Updater, possibly in an endless loop with a set of options. 
 
 ### Flags
 
-**--application-namespace *namespace***
+**--application-namespace *token* **
```

Comment:
need to remove the space near the end. `namespace` looks good so let's keep it. So it will end like `*token***`

We've seen formatting issues with a space.